package_manifest.yaml

```diff
@@ -592,7 +592,7 @@ foreman_client_packages:
 katello_packages:
   vars:
     copr_projects:
-      - "{{ hostvars['katello-copr'] }}"
+      - "{{ hostvars['katello-copr-with-el9'] }}"
```

Comment:
mh, this will build *all* of katello on el9, which we do not want yet (katello itself does not define ruby 3 compat yet)?

Reply:
That was a leftover from doing some testing builds of the other packages.